CHANGELOG.md

```diff
@@ -3,6 +3,34 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v0.4.3] - 2025-11-07
+
+Adjusting alignment filtering for both nanopore and illumina data to remove supplementary reads and secondary reads. The supplementary reads were rarely adding in artifacts to the final consensus sequence including snps and indels that only they contained. This lowers read counts slightly
+
+Also added the updates to the clair3 command including bumping the version, and the vcf filter found in Artic to add in the SNP allele frequency cutoff along with the high frameshift indel cutoff. There was also the addition of a RefCall filter to prevent those from being reported in the final report table (Note that they don't affect the consensus result at all, just the report visualization)
+
+Overall, the update should fix some rare artifacts to improve the accuracy of the final results
+
+### `Added`
+
+- Samtools sort nf-core module to replace samtools index module in illumina pipeline [PR #21](https://github.com/phac-nml/measeq/pull/21)
```

Comment:
I see Samtools and SAMtools, so it's not clear what's correct, but you should be consistent here (1 Samtools, 1 samtools).

Reply:
Samtools from their docs so I'll fix that up